single-location.php

```diff
@@ -2,46 +2,60 @@
 get_header();
 the_post();
 
+$address = ( isset( $post->ucf_location_address ) ) ? '<h2 class="h6 font-weight-normal">' . $post->ucf_location_address . '</h2>' : "";
 $lat_lng = $post->meta['ucf_location_lat_lng'];
 $lat_lng_string = ( $lat_lng ) ? $lat_lng["ucf_location_lat"] . "," . $lat_lng["ucf_location_lng"] : "";
 $events = ( isset( $post->meta['events_markup'] ) && ! empty( $post->meta['events_markup'] ) ) ? $post->meta['events_markup'] : null;
+$featured_image = wp_get_attachment_url( get_post_thumbnail_id($post->ID) );
+$location_fallback_image = "";
+
+if( ! empty( get_theme_mod_or_default( 'location_fallback_image' ) ) ) {
```

Comment:
Consider printing a single column and `<img>` tag, where just the `img src` is modified depending on what's available for the given location.  This way we don't have to maintain similar column and img markup in multiple places.

e.g.
```php
$featured_image = wp_get_attachment_url( get_post_thumbnail_id( $post->ID ) );
$location_fallback_image = get_theme_mod_or_default( 'location_fallback_image' );
$location_image = '';
if ( $featured_image ) {
	$location_image = $featured_image;
}
else if ( ! empty( $location_fallback_image ) ) {
	$location_image = $location_fallback_image;
}
?>
...
<div class="row mb-3">
	<?php if ( $location_image ) : ?>
	<div class="col-md-6">
		<img src="<?php echo $location_image; ?>" alt="" class="img-fluid mb-3">
	</div>
	<?php endif; ?>
</div>
```